src/nimble.nim

```diff
@@ -851,14 +851,20 @@ proc uninstall(options: Options) =
       raise newException(NimbleError, "Package not found")
 
     display("Checking", "reverse dependencies", priority = HighPriority)
-    var errors: seq[string] = @[]
     for pkg in pkgList:
       # Check whether any packages depend on the ones the user is trying to
       # uninstall.
       if options.uninstallRevDeps:
         getAllRevDeps(options, pkg, pkgsToDelete)
       else:
-        let revDeps = getRevDeps(options, pkg)
+        let
+          # Pkgs already processed for removal
```

Comment:
The word "processed" here is misleading, it implies that these packages have already been deleted but this isn't the case. Rephrase this to something like this: "Names of packages already scheduled for removal"

(Please use full words in comments instead of abbreviations, it makes it harder to read, same for `revDeps` below).